packages/filesystem/src/browser/file-tree/file-tree-widget.tsx

```diff
@@ -118,6 +118,15 @@ export class FileTreeWidget extends TreeWidget {
                 this.model.expandNode(containing);
             }
         }, 500);
+
+        let offset: number = 0;
+        if (event.clientY <= 90) {
+            offset = -20;
+        }
+        if (event.clientY >= 900) {
```

Comment:
Magic numbers, they seem to only work with 1080p-ish windows.

Or maybe I just miss what the numbers mean, but the changes have an effect on a fullscreen browser window, but not when I resize to a smaller window.

Reply:
To replace the magic number, maybe just a matter of finding the scrollable area size and computing the values from here?

`const bounds = event.currentTarget.getBoundingClientRect()` ?

Reply:
found a way, thanks